### PR TITLE
docs.ropensci.org guidance

### PR DESCRIPTION
**Pull request by maelle**
I'll add other pkgdown guidance but for now I want to make sure I'm writing correct stuff about the central builds.

@jeroen can you have a first look before I ask editors?

### CODE REVIEW

File: pkg_building.Rmd [hunk 1]
```diff
@@ -163,7 +163,14 @@ can be found at *link*".
...
 
-There are a few tips we'd like to underline here.
+There are a few elements we'd like to underline here.
+
+### Automatic deployment of the documentation website {docsropensci}
+
+You only need to worry about automatic deployment of your website until approval and transfer of your package repo to the ropensci organization; indeed, after that a pkgdown website will be built for your package after each push to the GitHub repo. You can find the status of these builds at `https://dev.ropensci.org/job/package_name`, e.g. [for `magick`](https://dev.ropensci.org/job/magick); and the website at `https://docs.ropensci.org/package_name`, e.g. [for `magick`](https://docs.ropensci.org/job/magick). The website build will use your pkgdown config file if you have one, except for the styling that will use the [`rotemplate` package](https://docs.ropensci.org/rotemplate/). Please report bugs, questions and feature requests about the central builds at https://github.com/ropensci/docs/ and about the template at https://github.com/ropensci/rotemplate/.
```
Comment (Bisaloo) [inline pkg_building.Rmd]: The link here seem to suffer from a copy/paste mistake (sorry if I'm overstepping here, I was just looking at open PR before I opened a new one)

```suggestion
You only need to worry about automatic deployment of your website until approval and transfer of your package repo to the ropensci organization; indeed, after that a pkgdown website will be built for your package after each push to the GitHub repo. You can find the status of these builds at `https://dev.ropensci.org/job/package_name`, e.g. [for `magick`](https://dev.ropensci.org/job/magick); and the website at `https://docs.ropensci.org/package_name`, e.g. [for `magick`](https://docs.ropensci.org/magick). The website build will use your pkgdown config file if you have one, except for the styling that will use the [`rotemplate` package](https://docs.ropensci.org/rotemplate/). Please report bugs, questions and feature requests about the central builds at https://github.com/ropensci/docs/ and about the template at https://github.com/ropensci/rotemplate/.
```

File: pkg_ci.Rmd [hunk 2]
```diff
@@ -97,10 +97,6 @@ If you run coverage on several CI services [the results will be merged](https://
...
+After transfer to rOpenSci's "ropensci" GitHub organization, a pkgdown website will be built for your package after each push to the GitHub repo. You can find the status of these builds at `https://dev.ropensci.org/job/package_name`, e.g. [for `magick`](https://dev.ropensci.org/job/magick); and the website at `https://docs.ropensci.org/package_name`, e.g. [for `magick`](https://docs.ropensci.org/job/magick). The website build will use your pkgdown config file if you have one, except for the styling that will use the [`rotemplate` package](https://docs.ropensci.org/rotemplate/). Please report bugs, questions and feature requests about the central builds at https://github.com/ropensci/docs/ and about the template at https://github.com/ropensci/rotemplate/.
```
Comment (Bisaloo) [inline pkg_ci.Rmd]: same here https://docs.ropensci.org/job/magick -> https://docs.ropensci.org/magick

File: pkg_building.Rmd (hunk 1 above)
Comment (maelle) [reply pkg_building.Rmd]: No overstepping, thanks a ton!

**Comment by jeroen** [comment]
Thanks, this sounds great.

I'm not sure if we should recommend to setup pkgdown via tic/travis while they are in the onboarding process. Doing so will advertise the package website on a custom url, and once they the onboarding has completed, hey have to remove their integration again and then update the website url everywhere they have advertised it. 

Other than that I really like it, thanks!

**Comment by maelle** [comment]
Thank you!

> I'm not sure if we should recommend to setup pkgdown via tic/travis while they are in the onboarding process. Doing so will advertise the package website on a custom url, and once they the onboarding has completed, hey have to remove their integration again and then update the website url everywhere they have advertised it.

Good point, I'll think about it. We do want them to have an "easy" way to have an up-to-date pkgdown website during development/review because reviewers will read it (a pkgdown website could be a good point of entry for a review), but I see the problem. Maybe I'll write a helper function to update links. :thinking:

**Comment by maelle** [comment]
:wave: @ropensci/editors any feedback on this PR, and any opinion on @jeroen's remark above? Thanks in advance! :smile_cat:

**Pull request merged**